### ISSUE TITLE
tf_level problem

### ISSUE THREAD

**Issue by Jigsawbg** [GyanD/codexffmpeg#126]
Hello Gyan

Since implementing `-tune uhq`, `-tf_level` and `-lookahead_level`, `ffmpeg` cannot encode the video correctly. I think the problem is in `tf_level`. I use the following settings for hevc_nvenc:
-tune 5
-tf_level 4
-lookahead_level 15
With these settings, the file is encoded, but after encoding is complete, the file is always `isTruncated` in MediaInfo and the usual statistics at the end of encoding is missing( I think `ffmepg` just crash silently ). I used `-report` and it shows that the last line is:
`[matroska @ 0000014b9ca75f00] Writing block of size 111 with pts 341292, dts 341167, duration 42 at relative offset 263359 in cluster at offset 159361861. TrackNumber 1, keyframe 0`

If I use `tf_level 0` then the encoding completes normally and file is not truncated.

I'm on Windows 10 pro 22H2 OS build 19045.4291
Nvidia RTX 3060Ti Founders Edition
Driver 551.86

The problem also exists with version 2024-04-15-git-5e380bcdb1

**Comment by GyanD** [comment]
This needs to be reported to ffmpeg at trac.ffmpeg.org

**Comment by Jigsawbg** [comment]
Ok, Thanks Gyan. I create ticket in trac.ffmpeg.org.

**Comment by Jigsawbg** [comment]
Gyan, sorry to bother you, but if I understand correctly the reason you closed the issue is that it cannot be reproduced.
Sorry, it's my fault for not providing the command line that is causing the problem. I will also correct the ticket description in trace.ffmpeg.org.
Actually, the problem is not in `tf_level 4` per se. The problem occurs when I use:
`-hwaccel cuda -hwaccel_output_format cuda` or `-hwaccel cuda` or `-hwaccel nvdec` or `-hwaccel dxva2` or `hwaccel auto`. . . I tested with these.
If I use `-c:v h264_cuvid` then no problem.

This is enough to recreate the problem:
`ffmpeg.exe -hwaccel auto -i input.m2ts -map 0:v:0 -c:v hevc_nvenc -bf 4 -tf_level 4 output.mkv`

That's how it works:
`ffmpeg.exe -c:v h264_cuvid -i input.m2ts -map 0:v:0 -c:v hevc_nvenc -bf 4 -tf_level 4 output.mkv`

The rest of the options that I use in the command line are not relevant to the problem, so I have omitted them in the example, I have already checked it.

Once again, I'm sorry!
